lib/ansible/modules/cloud/azure/azure_rm_trafficmanagerprofile.py

```diff
@@ -0,0 +1,451 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2018 Hai Cao, <t-haicao@microsoft.com> Yunge Zhu <yungez@microsoft.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: azure_rm_trafficmanagerprofile
+version_added: "2.7"
+short_description: Manage a Traffic Manager profile.
+description:
+    - Create, update and delete a Traffic Manager profile.
+
+options:
+    resource_group:
+        description:
+            - Name of a resource group where the Traffic Manager profile exists or will be created.
+        required: true
+    name:
+        description:
+            - Name of the Traffic Manager profile.
+        required: true
+    state:
+        description:
+            - Assert the state of the Traffic Manager profile. Use C(present) to create or update a Traffic Manager profile and C(absent) to delete it.
+        default: present
+        choices:
+            - absent
+            - present
+    location:
+        description:
+            - Valid azure location. Defaults to 'global' because in default public Azure cloud, Traffic Manager profile can only be deployed globally.
```

Comment:
Add the key `default: global` instead of just adding it to the description. You should be setting this as part of your argument spec as well (commented below).

Reply:
updated